plugins/@grouparoo/mysql/public/templates/destination/{{{id}}}.js.template

```diff
@@ -7,7 +7,9 @@ exports.default = async function buildConfig() {
       type: "mysql-export-records",
       appId: {{{appId}}}, // The ID of the App this Destination uses - e.g. `appId: "data_warehouse"`
       modelId: "mod_profiles", // The ID of the Grouparoo Model that this Destination will send
+      collection: "group", // What collection of Records should be sent?  "group" requires a groupId below, and "model" will sync all Records from "modelId"
       groupId: "...", // The ID of the group whose members you want to export - e.g. `groupId: "high_value_customers"`
+      syncMode: {{{syncMode}}}, // How should Grouparoo sync with this destination? Options: "sync", "additive", "enrich"
```

Comment:
Some destination templates were missing syncMode

Reply:
It doesn't look like `mysql` supports syncModes at all... 🤔